student.rb

```diff
@@ -9,6 +9,11 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     classroom.students.push(self) unless classroom.students.include?(self)
```

Comment:
**[OPTIONAL]**

- Hey :wave: Since you've implicitly added this rule in the method `classroom`, you don't need to add it here anymore. All you need here is just the instance variable which is the array you're pushing.